### DISCUSSION TITLE
Human resource with varying max hours per day

### DISCUSSION THREAD

**Discussion by tjheiska** [frePPLe/frepple#435] [Q&A] [answered]
Hi,

I have modeled human resource with default type and available calendar with working hours.
This works fine, but now I would like to add maximum hours per varying by the day. For example: operator is available from 9-5, but can only have maximum 4 hours of work in specific dates.
Can you help me with this? Thanks!

**Comment by jdetaeye** [comment, marked as answer]
Sounds like a bucketized resource with a maximum number of hours per day.
Instead of a constraint "mr X can do only 1 job at a time", it models a constraint "mr X has max 4 hours per day".

**Reply by tjheiska** [reply]
Thanks for a quick reply. I though it would work like that. Do you have or know any example how to do that? I changed the resource type to 'buckets' and added maximum calendar with bucket containing value 4.0. Now I don't get the tasks planned anymore.

**Reply by jdetaeye** [reply]
https://frepple.org/docs/current/user-guide/examples/resource/resource-type.php

**Reply by tjheiska** [reply]
Thanks again. Have seen that example before. Seems that my problem is the inheritance. I have resource hierarchy where 'Operator' is the top level resource, which is inherited by individual operators, which are further inherited by individual operators working at certain workstation. Changing the main 'Operator' to be bucketized seems to work. So I just need to figure out how that works for inheritance.
